### PR TITLE
Changing mitx to edx-platform and adding code_path param

### PR DESCRIPTION
**Pull request by jarv**
These changes may be already made on another branch, not sure

### CODE REVIEW

File: playbooks/roles/lms/tasks/main.yml [hunk 1]
```diff
@@ -58,15 +58,15 @@
...
 - name: install git and its recommends
   apt: pkg=git state=present install_recommends=yes 
   tags:
   - lms
   - cms
 
-- name: git checkout edx-platform repo into $app_base_dir
-  git: dest={{platform_code_dir}} repo={{lms_source_repo}}
+- name: git checkout edx-platform repo into app_base_dir
+  git: dest=$lms_code_dir repo=$lms_source_repo
```
Comment (jarv) [inline playbooks/roles/lms/tasks/main.yml:69]: I was having trouble using jinja syntax for these params, this seems to work ok.

**Comment by jbau** [comment]
I think @jrbl has done this in feature/jrbl/parameterized_git_target

**Comment by jbau** [comment]
maybe you two can figure out the review?

**Comment by jarv** [comment]
sounds good, I'll wait until he is ready to merge.

**Pull request closed without merging**